presto-docs/src/main/sphinx/connector/hana.rst

```diff
@@ -0,0 +1,100 @@
+====================
+HANA Connector
+====================
+
+The HANA connector allows querying and creating tables in an
+external HANA database. This can be used to join data between
+different systems like HANA and Hive, or between two different
+HANA instances.
+
+Configuration
+-------------
+
+To configure the HANA connector, create a catalog properties file
+in ``etc/catalog`` named, for example, ``hana.properties``, to
+mount the HANA connector as the ``hana`` catalog.
+Create the file with the following contents, replacing the
+connection properties as appropriate for your setup:
+
+.. code-block:: none
+
+    connector.name=hana
+    connection-url=jdbc:sap://[serverName[\instanceName][:portNumber]]
+    connection-user=root
+    connection-password=secret
+
+Multiple HANA Databases or Servers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The HANA connector can only access a single database within
+a HANA server. Thus, if you have multiple HANA databases,
+or want to connect to multiple instances of the HANA, you must configure
+multiple catalogs, one for each instance.
+
+To add another catalog, simply add another properties file to ``etc/catalog``
+with a different name (making sure it ends in ``.properties``). For example,
```

Comment:
Just commenting that this is a great explanation that the properties file name must end in .properties! I remember not understanding that I could have different catalogs using the same connector by creating different properties files when I started working with Presto.

No change suggested, I wanted to compliment this as good documentation.